script.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
```

Comment:
Why putting this file at project root level ?

Reply:
oups... I will remove it...